### PR TITLE
Set minio image to use minio user

### PR DESCRIPTION
**Pull request by denhamparry**
- minio image requires minio user

### CODE REVIEW

File: images/minio/config/template.apko.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+contents:
+  packages:
+    - minio
+    - minio-oci-entrypoint
+    - busybox
```
Comment (joshrwolf) [inline images/minio/config/template.apko.yaml:5]:
```suggestion
```

you can remove this since it's a package dependency now

Comment (denhamparry) [reply images/minio/config/template.apko.yaml:5]: I think I can remove both, as the entry script was just creating a user that we're already doing

File: images/minio/config/template.apko.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  groups:
+    - groupname: minio
+      gid: 1000
+  users:
+    - username: minio
+      uid: 1000
+  run-as: 1000
+
+entrypoint:
+  command: /usr/bin/docker-entrypoint.sh
```
Comment (joshrwolf) [inline images/minio/config/template.apko.yaml:15]:
```suggestion
  command: /usr/bin/minio
```

File: images/minio/config/template.apko.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - minio
+
+accounts:
+  groups:
+    - groupname: minio
+      gid: 1000
+  users:
+    - username: minio
+      uid: 1000
+  run-as: 1000
```
Comment (joshrwolf) [inline images/minio/config/template.apko.yaml:12]: did you pick this? or did it come from somewhere? not disagreeing just asking for context

**Comment by joshrwolf** [comment]
leaving for future ref, upstream uses [this](https://explore.ggcr.dev/layers/quay.io/minio/minio@sha256:7b40c61317c661d5d87428a287cd3f9c588ad4177b3ae50107be040359837485/usr/bin/docker-entrypoint.sh) entrypoint, which appears to be deprecated but still left in the image for whatever reason.

this approach simply defines the user as `1000` in the oci config, and makes the necessary adjustments to the `apko` file to ensure that user exists.

File: images/minio/config/template.apko.yaml (hunk 3 above)
Comment (denhamparry) [reply images/minio/config/template.apko.yaml:12]: https://github.com/minio/minio/blob/master/dockerscripts/docker-entrypoint.sh
I looked at the file and could see it setup the user with the uid and gid.
I stuck with it as I searched and saw some other images do it.

**Review by imjasonh — APPROVED** [review]

**Comment by denhamparry** [comment]
Confirmation that this isn't required. Minio is working as expected so doesn't require this PR. Closing the PR.

**Pull request closed without merging**